drivers/SmartThings/zigbee-window-treatment/profiles/aqara-curtain.yml

```diff
@@ -1,4 +1,4 @@
-name: window-treatment-aqara
+name: aqara-curtain

```

Comment:
once a profile has been created, it cannot be moved/renamed as this can create "orphaned" devices that are unaware of the change

Reply:
Okay. I will change the name as below.
- Curtain controller: window-treatment-aqara.yml
- Roller Shade Controller: window-treatment-aqara-roller-shade-rotate.yml